docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide explains how to access the monitoring stack. Lokomotive ships monitoring stack in the form of prometheus-operator component. Learn more about the prometheus-operator component and its parameters [here](../configuration-reference/components/prometheus-operator.md).
+
+## Prerequisites
+
+- Deploy the prometheus-operator component in `monitoring` namespace. If different namespace used in the configuration then change the subsequent commands accordingly, `kubectl -n <namespace> ...`.
```

Comment:
>Deploy the prometheus-operator component in `monitoring` namespace.

There is a missing article here.

>If different namespace used in the configuration then change the subsequent commands accordingly, `kubectl -n <namespace> ...`.

- Missing article before "different".
- Missing verb before "used" (should be "is used").
- There should be a comma after "configuration" (since the conditional is an introductory phrase) according to our style guide.
- There should not be a "then" after a conditional statement according to our style guide.

I would write the sentence as follows:

>This guide assumes the prometheus-operator component is deployed in a namespace called `monitoring`. If a different namespace is used, replace `monitoring` with that namespace throughout this guide.